docs/aerodromes/classd/Moorabbin.md

```diff
@@ -81,13 +116,88 @@ Aircraft who will transit Class G airspace on climb into CTA will be **cleared t
+- On completion of circuit training, the pilot will nominate their landing HLS during the airborne call and report downwind before transitioning to the arrival procedure.
+
+!!! example
+    **RJB**:: "MB Gnd, RJB, At the southern apron, with information november, request circuits"
+    **MB SMC**: "RJB, cleared for circuit, Eastern Grass, report airborne"      
+    **RJB**: "Cleared for circuits, Eastern Grass, RJB"  
+    *(Before departure)*  
+    **RJB**: "Moorabbin Tower, RJB, airborne"  
+    **MB ADC**: "RJB"
+
+#### Night Circuits
+Night operations must comply with fixed wing circuits and altitudes.
+
+### Departures
+
+Day VFR helicopter departures must remain at `A007`. Helicopters must nominate a aiming point or HLS on first contact with MB SMC.
+
+1. **Aiming Points**:
+   - Direct helicopters departing from the **Southern Aiming Point** to hold short of **TWY A**.
+   - Direct helicopters departing from the **Northern Aiming Point** to hold short of **TWY G**.
+
+2. **Departure Procedures**:
+    - Ensure departures remain clear of duty runways and provide instructions to:
+        - **Pass east of the tower**.
+        - **Track**:
+            - For **northbound departures**: Track north until clear of the tower.
+            - For **southbound departures**: Track south until clear of airport buildings.
+
+<figure markdown>
+![Aiming Locations](img/mbhelidep.png){ width="700" }
+<figcaption>Aiming Locations</figcaption>
+</figure>
+
+### Arrivals
+
+- ML TCU will heads-up coordinate arrivals/overfliers from Class C to MB ADC.  
+
+- **From the East**:
+    - Helicopters arriving from the east should be instructed to fly a close base leg and pass over the duty threshold at **not below `A003`**.
+    - For duty runways **13** or **17**, helicopters arriving for the northern or main aprons may be instructed to overfly the duty threshold at **not below `A003`** for a short right circuit.
+    - For **circuits**, helicopters arriving may directly join a leg of the circuit.
+
+- **From the West**:
+    - For **southerly approaches** full stop: Instruct helicopters to remain clear of duty runways, track east of the tower, and proceed directly to the **Southern HLS** or southern aiming point.
+    - For **northerly approaches** full stop: Instruct helicopters to remain clear of duty runways, track east of the tower, and proceed to the northern or main aprons.
+    - For **circuits** have the aircraft fly overhead the tower and report. Instruct the aircraft to join a leg of the circuit.
+
+!!! example
+    A helicopter, RJB, is looking to join the circuits from the west at Cerberus, with runway 17L/R active.
+    **RJB**: "MB Twr, RJB, Heli R44, Cerberus 700, inbound, with november, for circuits"
+    **MB ADC**: "RJB, MB Twr, report overhead tower"
+    **RJB**: "RJB overhead"
+    **MB ADC**: "RJB, join downwind runway 17R"
+    **RJB**: "join downwind runway, 17R"
```

Comment:
```suggestion
    VH-RJB, an R44 helicopter, is looking to join the circuit from the west at Cerberus, with runway 17L/R active.  
    **RJB**: "Moorabbin Tower, Helicopter RJB, Cerberus, A007, inbound, with november, for circuits"  
    **MB ADC**: "RJB, Moorabbin Tower, report overhead the tower"  
    **RJB**: "Wilco, RJB"  
    
    **RJB**: "RJB overhead the tower"
    **MB ADC**: "RJB, join downwind runway 17R"
    **RJB**: "Join downwind runway, 17R"
```

From your experience, is a 'cleared visual approach' required here, or is the absence of a level assignment sufficient to allow the pilot to descend at their discretion? I can imagine online pilots either descending early and causing conflict with other circuit traffic while overhead, or not descending at all.